filter-ops.json

```diff
@@ -2,6 +2,7 @@
     "projectSchema": [
         "id",
         "name",
-        "description"
+        "description",
+        "mode"
```

Comment:
Not enough to just update, we also need to expose it here since the project schema breaks by default